gradoop-temporal/src/main/java/org/gradoop/temporal/model/impl/operators/keyedgrouping/TemporalGroupingKeys.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright © 2014 - 2019 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.temporal.model.impl.operators.keyedgrouping;
+
+import org.apache.flink.api.java.tuple.Tuple2;
+import org.gradoop.flink.model.api.functions.KeyFunction;
+import org.gradoop.temporal.model.api.functions.TimeDimension;
+import org.gradoop.temporal.model.impl.operators.keyedgrouping.keys.DurationKeyFunction;
+import org.gradoop.temporal.model.impl.operators.keyedgrouping.keys.TimeIntervalKeyFunction;
+import org.gradoop.temporal.model.impl.operators.keyedgrouping.keys.TimeStampKeyFunction;
+import org.gradoop.temporal.model.impl.pojo.TemporalElement;
+
+import java.time.temporal.TemporalField;
+import java.time.temporal.TemporalUnit;
+
+/**
+ * A factory class for creating instances of commonly used grouping key functions for grouping
+ * on temporal attributes.
+ */
+public class TemporalGroupingKeys {
+  /**
+   * No instances of this class are needed.
+   */
+  private TemporalGroupingKeys() {
+  }
+
+  /**
+   * Group by duration of a time interval. The duration will be calculated in a certain unit.
+   *
+   * @param interval The time interval to get the duration of.
+   * @param timeUnit The unit in which the duration is measured.
+   * @param <T> The type of the elements to group.
+   * @return The grouping key function extracting the duration.
+   */
```

Comment:
Add `@see DurationKeyFunction` to Javadoc and similarly for the other functions.